modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -751,13 +752,13 @@ private void applyOperations(Collection<Operation> ops, HybridTimestamp opTs) th
 
         try (WriteBatch batch = new WriteBatch()) {
             for (Operation op : ops) {
-                byte[] key = op.key();
+                byte @Nullable [] key = op.key() == null ? null : toByteArray(op.key());
```

Comment:
Is nullable annotation required here?

Reply:
I think yes, it became clearer to me when debugging.